server/graphql/v2/object/Order.js

```diff
@@ -142,6 +145,23 @@ export const Order = new GraphQLObjectType({
           }
         },
       },
+      membership: {
+        type: MemberOf,
+        description:
+          'This represents a MemberOf relationship (ie: Collective backed by an Individual) attached to the Order.',
+        async resolve(order) {
+          const fromAccount = await order.getFromCollective();
+          const toAccount = await order.getCollective();
+
+          return models.Member.findOne({
+            where: {
+              MemberCollectiveId: fromAccount.id,
+              CollectiveId: toAccount.id,
+              role: roles.BACKER,
```

Comment:
Members are also grouped by tier. To be accurate here we should to:

```suggestion
              role: roles.BACKER,
              TierId: order.TierId,
```